src/ir/comp.rs

```diff
@@ -825,9 +825,15 @@ impl CompFields {
                     }
 
                     anon_field_counter += 1;
-                    let generated_name =
-                        format!("__bindgen_anon_{}", anon_field_counter);
-                    *name = Some(generated_name);
+                    if let Some(ref prefix) = ctx.options().anon_fields_prefix {
+                        *name =
+                            Some(format!("{}{}", prefix, anon_field_counter));
+                    } else {
+                        *name = Some(format!(
+                            "__bindgen_anon_{}",
+                            anon_field_counter
+                        ));
+                    }
```

Comment:
Would using `unwrap_or` be better than `if`-`else` here ? I admit that the need to satisfy the borrow-checker makes it not quite as attractive as I had hoped.
```rust
*name = Some(format!(
    "{}{}",
    ctx.options()
        .anon_fields_prefix
        .as_ref()
        .map(|s| s.as_str())
        .unwrap_or("__bindgen_anon_"),
    anon_field_counter
));
```

This does not really apply if you use my other suggestion.